### PR TITLE
German translation (i18n)

### PR DESCRIPTION
**Pull request by EarMaster**
Added de.json

### What does it do?

Added german translations for the i18n plugin.

### Why is it needed?

There weren't any yet.

### How to test it?

Learn german (optional), set admin panel locale to Deutsch. Open the i18n plugin settings or the content-type editor or the content-manager. Enjoy.

### Related issue(s)/PR(s)

None

### CODE REVIEW

**Comment by gu-stav** [comment]
@EarMaster Thanks for your contribution. I'll have a look into this PR tomorrow and review it. ✨

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "CMEditViewCopyLocale.copy-failure": "Beim Kopieren der Übersetzungen ist ein Fehler aufgetreten",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:2]:
```suggestion
  "CMEditViewCopyLocale.copy-failure": "Beim Kopieren der Übersetzung ist ein Fehler aufgetreten",
```

I think this needs to be singular.

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  "CMEditViewCopyLocale.copy-success": "Übersetzungen kopiert",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:3]:
```suggestion
  "CMEditViewCopyLocale.copy-success": "Übersetzung kopiert",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  "CheckboxConfirmation.Modal.body": "Möchtest du die Lokalisierung deaktivieren?",
+  "CheckboxConfirmation.Modal.button-confirm": "Ja, deaktivieren",
+  "CheckboxConfirmation.Modal.content": "Wenn Du die Lokalisierung deaktivierst, werden alle Ihre Inhalte mit Ausnahme des Inhalts, der Ihrem Standardgebietsschema zugeordnet ist (sofern vorhanden), gelöscht.",
+  "Field.localized": "Dieser Wert ist einzigartig für die ausgewählte Übersetzung",
+  "Field.not-localized": "Dieser Wert ist für alle Übersetzungen gleich",
+  "Settings.list.actions.add": "Neue Übersetzung hinzufügen",
+  "Settings.list.actions.delete": "Übersetzung löschen",
+  "Settings.list.actions.deleteAdditionalInfos": "Dies wird die aktive Übersetzung löschen.",
+  "Settings.list.actions.edit": "Übersetzung bearbeiten",
+  "Settings.list.description": "Konfiguriere die Einstellungen für das Internationalisierung Plugin",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:16]:
```suggestion
  "Settings.list.description": "Einstellungen für das Internationalisierung Plugin anpassen",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.list.empty.description": "Das ist ein unerwartetes Verhalten. Möglicherweise hast du die Datenbank manuell modifiziert. Stelle sicher, dass mindestens eine Übersetzung in der Datenbank existiert, um Strapi nutzen zu können",
+  "Settings.list.empty.title": "Keine Übersetzungen vorhanden.",
+  "Settings.locales.default": "Standard",
+  "Settings.locales.list.sort.default": "Sortieren nach Standard-Übersetzung",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:20]:
```suggestion
  "Settings.locales.list.sort.default": "Sortierung nach Standard-Übersetzung",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.list.sort.displayName": "Sortieren nach Anzeigename",
+  "Settings.locales.list.sort.id": "Sortieren nach ID",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:22]:
```suggestion
  "Settings.locales.list.sort.id": "Sortierung nach ID",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.advanced": "Fortgeschrittene Einstellungen",
+  "Settings.locales.modal.advanced.setAsDefault": "Als Standard-Übersetzung verwenden",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:24]:
```suggestion
  "Settings.locales.modal.advanced.setAsDefault": "Als Standardsprache verwenden",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.advanced.setAsDefault.hint": "Es muss eine Standard-Übersetzung geben, ändere diese, indem eine andere Übersetzung als Standard-Übersetzung gewählt wird.",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:25]:
```suggestion
  "Settings.locales.modal.advanced.setAsDefault.hint": "Eine Standardsprache muss ausgewählt sein.",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.advanced.settings": "Einstellungen",
+  "Settings.locales.modal.base": "Grundeinstellungen",
+  "Settings.locales.modal.create.alreadyExist": "Diese Übersetzung existiert bereits",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:28]:
```suggestion
  "Settings.locales.modal.create.alreadyExist": "Diese Sprache existiert bereits",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.create.defaultLocales.loading": "Verfügbare Übersetzungen werden geladen...",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:29]:
```suggestion
  "Settings.locales.modal.create.defaultLocales.loading": "Verfügbare Sprachen werden geladen...",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.create.success": "Übersetzung hinzugefügt",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:30]:
```suggestion
  "Settings.locales.modal.create.success": "Sprache hinzugefügt",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.create.tab.label": "Wechseln zwischen den Grundeinstellungen und den fortgeschrittenen Einstellungen",
+  "Settings.locales.modal.delete.confirm": "Ja, löschen",
+  "Settings.locales.modal.delete.message": "Das Löschen dieser Übersetzung wird allen Inhalt in dieser Übersetzung löschen. Um Inhalte zu behalte, kopiere sie zuerst in andere Übersetzungen.",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:33]:
```suggestion
  "Settings.locales.modal.delete.message": "Beim Löschen dieser Sprache werden allen zugeordneten Inhalt ebenfalls gelöscht. Um Inhalte zu behalten müssen diese zunächst in eine andere Sprache kopiert werden.",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.delete.secondMessage": "Übersetzung löschen?",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:34]:
```suggestion
  "Settings.locales.modal.delete.secondMessage": "Sprache löschen?",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.delete.success": "Übersetzung gelöscht",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:35]:
```suggestion
  "Settings.locales.modal.delete.success": "Sprache gelöscht",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.edit.confirmation": "Fertig",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:36]:
```suggestion
  "Settings.locales.modal.edit.confirmation": "Abschließen",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.edit.locales.label": "Übersetzung",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:37]:
```suggestion
  "Settings.locales.modal.edit.locales.label": "Sprache",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.edit.success": "Übersetzung bearbeitet",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:38]:
```suggestion
  "Settings.locales.modal.edit.success": "Sprache bearbeitet",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.edit.tab.label": "Wechseln zwischen den Grundeinstellungen und den fortgeschrittenen Einstellungen",
+  "Settings.locales.modal.locales.displayName": "Anzeigename",
+  "Settings.locales.modal.locales.displayName.description": "Die Übersetzungen wird unter diesem Namen im Admin-Panel angezeigt",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:41]:
```suggestion
  "Settings.locales.modal.locales.displayName.description": "Unter diesem Namen wird die Sprache im Admin-Panel angezeigt",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.locales.displayName.error": "Der Anzeigename kann max. 50 Zeichen haben.",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:42]:
```suggestion
  "Settings.locales.modal.locales.displayName.error": "Der Anzeigename kann aus max. 50 Zeichen bestehen.",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+  "components.Select.locales.not-available": "Kein Inhalt verfügbar",
+  "plugin.description.long": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.description.short": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.name": "Internationalisierung",
+  "plugin.schema.i18n.ensure-unique-localization": "Einzigartige Felder müssen lokalisiert sein",
+  "plugin.schema.i18n.localized.description": "Ermöglicht es Inhalte in verscheidenen Übersetzungen zu pflegen",
+  "plugin.schema.i18n.localized.description-content-type": "Ermöglicht es Inhalte in verscheidenen Übersetzungen zu pflegen",
+  "plugin.schema.i18n.localized.description-field": "Das Feld kann verschiedene Werte in jeder Übersetung haben",
+  "plugin.schema.i18n.localized.label": "Lokalisierung für diesen Content-Typ aktivieren",
+  "plugin.schema.i18n.localized.label-content-type": "Lokalisierung für diesen Content-Typ aktivieren",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:62]:
```suggestion
  "plugin.schema.i18n.localized.label-content-type": "Lokalisierung für diesen Inhaltstyp aktivieren",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+  "actions.select-locale": "Übersetzung auswählen",
+  "components.Select.locales.not-available": "Kein Inhalt verfügbar",
+  "plugin.description.long": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.description.short": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.name": "Internationalisierung",
+  "plugin.schema.i18n.ensure-unique-localization": "Einzigartige Felder müssen lokalisiert sein",
+  "plugin.schema.i18n.localized.description": "Ermöglicht es Inhalte in verscheidenen Übersetzungen zu pflegen",
+  "plugin.schema.i18n.localized.description-content-type": "Ermöglicht es Inhalte in verscheidenen Übersetzungen zu pflegen",
+  "plugin.schema.i18n.localized.description-field": "Das Feld kann verschiedene Werte in jeder Übersetung haben",
+  "plugin.schema.i18n.localized.label": "Lokalisierung für diesen Content-Typ aktivieren",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:61]:
```suggestion
  "plugin.schema.i18n.localized.label": "Lokalisierung für diesen Inhaltstyp aktivieren",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 21]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.permissions.read.denied.description": "Um dies zu lesen setze dich bitte mit dem Administrator in Verbindung.",
+  "Settings.permissions.read.denied.title": "Kein Zugriff auf diesen Inhalt.",
+  "actions.select-locale": "Übersetzung auswählen",
+  "components.Select.locales.not-available": "Kein Inhalt verfügbar",
+  "plugin.description.long": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.description.short": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.name": "Internationalisierung",
+  "plugin.schema.i18n.ensure-unique-localization": "Einzigartige Felder müssen lokalisiert sein",
+  "plugin.schema.i18n.localized.description": "Ermöglicht es Inhalte in verscheidenen Übersetzungen zu pflegen",
+  "plugin.schema.i18n.localized.description-content-type": "Ermöglicht es Inhalte in verscheidenen Übersetzungen zu pflegen",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:59]:
```suggestion
  "plugin.schema.i18n.localized.description-content-type": "Ermöglicht es Inhalte in verscheidenen Sprachen zu pflegen",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 22]
```diff
@@ -0,0 +1,64 @@
...
+  "plugin.schema.i18n.localized.description-field": "Das Feld kann verschiedene Werte in jeder Übersetung haben",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:60]:
```suggestion
  "plugin.schema.i18n.localized.description-field": "Das Feld kann verschiedene Werte in jeder Sprache haben",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 23]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.permissions.loading": "Berechtigungen werden geladen",
+  "Settings.permissions.read.denied.description": "Um dies zu lesen setze dich bitte mit dem Administrator in Verbindung.",
+  "Settings.permissions.read.denied.title": "Kein Zugriff auf diesen Inhalt.",
+  "actions.select-locale": "Übersetzung auswählen",
+  "components.Select.locales.not-available": "Kein Inhalt verfügbar",
+  "plugin.description.long": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.description.short": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
+  "plugin.name": "Internationalisierung",
+  "plugin.schema.i18n.ensure-unique-localization": "Einzigartige Felder müssen lokalisiert sein",
+  "plugin.schema.i18n.localized.description": "Ermöglicht es Inhalte in verscheidenen Übersetzungen zu pflegen",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:58]:
```suggestion
  "plugin.schema.i18n.localized.description": "Ermöglicht es Inhalte in verscheidenen Sprachen zu pflegen",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 24]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.locales.loaded": "Übersetzungen geladen.",
+  "Settings.locales.modal.title": "Einstellungen",
+  "Settings.locales.row.default-locale": "Standard-Übersetzung",
+  "Settings.locales.row.displayName": "Anzeigename",
+  "Settings.locales.row.id": "ID",
+  "Settings.permissions.loading": "Berechtigungen werden geladen",
+  "Settings.permissions.read.denied.description": "Um dies zu lesen setze dich bitte mit dem Administrator in Verbindung.",
+  "Settings.permissions.read.denied.title": "Kein Zugriff auf diesen Inhalt.",
+  "actions.select-locale": "Übersetzung auswählen",
+  "components.Select.locales.not-available": "Kein Inhalt verfügbar",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:53]:
```suggestion
  "components.Select.locales.not-available": "Keine Inhalte verfügbar",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 25]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.locales.displayName.description": "Die Übersetzungen wird unter diesem Namen im Admin-Panel angezeigt",
+  "Settings.locales.modal.locales.displayName.error": "Der Anzeigename kann max. 50 Zeichen haben.",
+  "Settings.locales.modal.locales.label": "Übersetzung",
+  "Settings.locales.modal.locales.loaded": "Übersetzungen geladen.",
+  "Settings.locales.modal.title": "Einstellungen",
+  "Settings.locales.row.default-locale": "Standard-Übersetzung",
+  "Settings.locales.row.displayName": "Anzeigename",
+  "Settings.locales.row.id": "ID",
+  "Settings.permissions.loading": "Berechtigungen werden geladen",
+  "Settings.permissions.read.denied.description": "Um dies zu lesen setze dich bitte mit dem Administrator in Verbindung.",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:50]:
```suggestion
  "Settings.permissions.read.denied.description": "Um dieses Feld anzuzeigen, setze dich bitte mit dem Administrator in Verbindung.",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 26]
```diff
@@ -0,0 +1,64 @@
...
+  "CheckboxConfirmation.Modal.button-confirm": "Ja, deaktivieren",
+  "CheckboxConfirmation.Modal.content": "Wenn Du die Lokalisierung deaktivierst, werden alle Ihre Inhalte mit Ausnahme des Inhalts, der Ihrem Standardgebietsschema zugeordnet ist (sofern vorhanden), gelöscht.",
+  "Field.localized": "Dieser Wert ist einzigartig für die ausgewählte Übersetzung",
+  "Field.not-localized": "Dieser Wert ist für alle Übersetzungen gleich",
+  "Settings.list.actions.add": "Neue Übersetzung hinzufügen",
+  "Settings.list.actions.delete": "Übersetzung löschen",
+  "Settings.list.actions.deleteAdditionalInfos": "Dies wird die aktive Übersetzung löschen.",
+  "Settings.list.actions.edit": "Übersetzung bearbeiten",
+  "Settings.list.description": "Konfiguriere die Einstellungen für das Internationalisierung Plugin",
+  "Settings.list.empty.description": "Das ist ein unerwartetes Verhalten. Möglicherweise hast du die Datenbank manuell modifiziert. Stelle sicher, dass mindestens eine Übersetzung in der Datenbank existiert, um Strapi nutzen zu können",
```
Comment (gu-stav) [inline packages/plugins/i18n/admin/src/translations/de.json:17]:
```suggestion
  "Settings.list.empty.description": "Das ist ein unerwartetes Verhalten. Möglicherweise hast du die Datenbank manuell modifiziert. Stelle sicher, dass mindestens eine Sprache in der Datenbank existiert, um Strapi nutzen zu können",
```

**Review by gu-stav — CHANGES_REQUESTED** [review]
I'm very sorry for the amount of suggestions. I think it's quite complicated to translate this part, because the word "locale" doesn't always have the same meaning in German. I tried to distinguish between "language" (e.g. create a language) and "translation" (e.g. create a language).

This makes it possible to create a language (Sprache), which then is used to create translations "Übersetzung".

What do you think about that? Thank you for your contribution!

**Comment by EarMaster** [comment]
> I'm very sorry for the amount of suggestions. I think it's quite complicated to translate this part, because the word "locale" doesn't always have the same meaning in German. I tried to distinguish between "language" (e.g. create a language) and "translation" (e.g. create a language).
> 
> This makes it possible to create a language (Sprache), which then is used to create translations "Übersetzung".
> 
> What do you think about that? Thank you for your contribution!

My intention with using "Übersetzung" instead of "Sprache" was to differentiate between "base languages" (like "German", "Spanish") and what is called locale in English (which could differ to "Swiss German", "Mexikan Spanish"). The literal translation for "locale" would be "Gebietsschema" which - in my opinion - doesn't quite hits the nail so I resorted to "Übersetzung".

But I am fine with your changes also. Is there a way to accept them all at once?

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 23 above)
Comment (NicoHof) [inline packages/plugins/i18n/admin/src/translations/de.json:58]:
```suggestion
  "plugin.schema.i18n.localized.description": "Ermöglicht es Inhalte in verschiedenen Sprachen zu pflegen",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 27]
```diff
@@ -0,0 +1,64 @@
...
+  "CMEditViewCopyLocale.submit-text": "Ja, ausfüllen",
+  "CMListView.popover.display-locales.label": "Übersetzungen anzeigen",
+  "CheckboxConfirmation.Modal.body": "Möchtest du die Lokalisierung deaktivieren?",
+  "CheckboxConfirmation.Modal.button-confirm": "Ja, deaktivieren",
+  "CheckboxConfirmation.Modal.content": "Wenn Du die Lokalisierung deaktivierst, werden alle Ihre Inhalte mit Ausnahme des Inhalts, der Ihrem Standardgebietsschema zugeordnet ist (sofern vorhanden), gelöscht.",
+  "Field.localized": "Dieser Wert ist einzigartig für die ausgewählte Übersetzung",
+  "Field.not-localized": "Dieser Wert ist für alle Übersetzungen gleich",
+  "Settings.list.actions.add": "Neue Übersetzung hinzufügen",
+  "Settings.list.actions.delete": "Übersetzung löschen",
+  "Settings.list.actions.deleteAdditionalInfos": "Dies wird die aktive Übersetzung löschen.",
```
Comment (NicoHof) [inline packages/plugins/i18n/admin/src/translations/de.json:14]:
```suggestion
  "Settings.list.actions.deleteAdditionalInfos": "Dies wird die aktive Übersetzung löschen <em>(aus der Internationalisierung)</em>",
```

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 3 above)
Comment (NicoHof) [reply packages/plugins/i18n/admin/src/translations/de.json:16]:
```suggestion
  "Settings.list.description": "Einstellungen für das Internationalisierung-Plugin anpassen",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 28]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.list.actions.edit": "Übersetzung bearbeiten",
+  "Settings.list.description": "Konfiguriere die Einstellungen für das Internationalisierung Plugin",
+  "Settings.list.empty.description": "Das ist ein unerwartetes Verhalten. Möglicherweise hast du die Datenbank manuell modifiziert. Stelle sicher, dass mindestens eine Übersetzung in der Datenbank existiert, um Strapi nutzen zu können",
+  "Settings.list.empty.title": "Keine Übersetzungen vorhanden.",
```
Comment (NicoHof) [inline packages/plugins/i18n/admin/src/translations/de.json:18]:
```suggestion
  "Settings.list.empty.title": "Keine Übersetzungen vorhanden",
```

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 7 above)
Comment (NicoHof) [reply packages/plugins/i18n/admin/src/translations/de.json:25]:
```suggestion
  "Settings.locales.modal.advanced.setAsDefault.hint": "Eine Standardsprache muss ausgewählt sein",
```

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 11 above)
Comment (NicoHof) [reply packages/plugins/i18n/admin/src/translations/de.json:33]:
```suggestion
  "Settings.locales.modal.delete.message": "Beim Löschen dieser Sprache werden alle zugeordneten Inhalte ebenfalls gelöscht. Um Inhalte zu behalten müssen diese zunächst in eine andere Sprache kopiert werden",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 29]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.delete.success": "Übersetzung gelöscht",
+  "Settings.locales.modal.edit.confirmation": "Fertig",
+  "Settings.locales.modal.edit.locales.label": "Übersetzung",
+  "Settings.locales.modal.edit.success": "Übersetzung bearbeitet",
+  "Settings.locales.modal.edit.tab.label": "Wechseln zwischen den Grundeinstellungen und den fortgeschrittenen Einstellungen",
+  "Settings.locales.modal.locales.displayName": "Anzeigename",
+  "Settings.locales.modal.locales.displayName.description": "Die Übersetzungen wird unter diesem Namen im Admin-Panel angezeigt",
+  "Settings.locales.modal.locales.displayName.error": "Der Anzeigename kann max. 50 Zeichen haben.",
+  "Settings.locales.modal.locales.label": "Übersetzung",
+  "Settings.locales.modal.locales.loaded": "Übersetzungen geladen.",
```
Comment (NicoHof) [inline packages/plugins/i18n/admin/src/translations/de.json:44]:
```suggestion
  "Settings.locales.modal.locales.loaded": "Übersetzungen geladen",
```

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 25 above)
Comment (NicoHof) [reply packages/plugins/i18n/admin/src/translations/de.json:50]:
```suggestion
  "Settings.permissions.read.denied.description": "Um dieses Feld anzuzeigen, setze dich bitte mit dem Administrator in Verbindung",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 30]
```diff
@@ -0,0 +1,64 @@
...
+  "Settings.locales.modal.title": "Einstellungen",
+  "Settings.locales.row.default-locale": "Standard-Übersetzung",
+  "Settings.locales.row.displayName": "Anzeigename",
+  "Settings.locales.row.id": "ID",
+  "Settings.permissions.loading": "Berechtigungen werden geladen",
+  "Settings.permissions.read.denied.description": "Um dies zu lesen setze dich bitte mit dem Administrator in Verbindung.",
+  "Settings.permissions.read.denied.title": "Kein Zugriff auf diesen Inhalt.",
```
Comment (NicoHof) [inline packages/plugins/i18n/admin/src/translations/de.json:51]:
```suggestion
  "Settings.permissions.read.denied.title": "Kein Zugriff auf diesen Inhalt",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 31]
```diff
@@ -0,0 +1,64 @@
...
+  "actions.select-locale": "Übersetzung auswählen",
+  "components.Select.locales.not-available": "Kein Inhalt verfügbar",
+  "plugin.description.long": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
```
Comment (NicoHof) [inline packages/plugins/i18n/admin/src/translations/de.json:54]:
```suggestion
  "plugin.description.long": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin-Panel als auch über die API",
```

File: packages/plugins/i18n/admin/src/translations/de.json [hunk 32]
```diff
@@ -0,0 +1,64 @@
...
+  "plugin.description.short": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin Panel als auch über die API.",
```
Comment (NicoHof) [inline packages/plugins/i18n/admin/src/translations/de.json:55]:
```suggestion
  "plugin.description.short": "Dieses Plugin ermöglicht es, Inhalte in verschiedenen Sprachen zu erstellen, zu lesen und zu aktualisieren, sowohl über das Admin-Panel als auch über die API",
```

**Review by NicoHof — COMMENTED** [review]
Hi! I added some suggestions. (fixed typos, incorrect sentence).
I also noticed there is an inconsistence with fullstops. Sometimes there is one, sometimes not. (Also in the original en file)
I added suggestions to remove all of them but since they are also there in the english file feel free to keep them.

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 27 above)
Comment (EarMaster) [reply packages/plugins/i18n/admin/src/translations/de.json:14]: I know I omitted the last part here, but to be honest, I don't get it. Of course I am not deleting French from it's existence, but I am deleting it from the i18n of the current strapi instance. Is there a reason why this is specifically stated?

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 14 above)
Comment (EarMaster) [reply packages/plugins/i18n/admin/src/translations/de.json:36]: Are you sure "Abschließen" is in line with the rest of the dialog confirmation labels?

**Comment by EarMaster** [comment]
@NicoHof I agree with you. I tried to stick to the original file, but you may be right, it might be better to just remove all the fullstops.
@gu-stav  I have accepted some of your requested changes.

Maybe both of you want to provide an opinion on how to translate "locale" (see my [comment](https://github.com/strapi/strapi/pull/12654#issuecomment-1050749522) above).

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 27 above)
Comment (NicoHof) [reply packages/plugins/i18n/admin/src/translations/de.json:14]: TBH I am not too sure why its stated. But maybe it makes sense to keep it though - probably it is there for some reason.

**Comment by gu-stav** [comment]
@EarMaster Great, thanks. I'll give it another round of review tomorrow.

**Comment by NicoHof** [comment]
> @NicoHof I agree with you. I tried to stick to the original file, but you may be right, it might be better to just remove all the fullstops. @gu-stav I have accepted some of your requested changes.
> 
> Maybe both of you want to provide an opinion on how to translate "locale" (see my [comment](https://github.com/strapi/strapi/pull/12654#issuecomment-1050749522) above).



> 

Hi @EarMaster. I think I agree with the idea of "Sprache" instead of "Übersetzung" in this i18n settings. For me it feels more relatable to create a language (Sprache) in the settings. And with this created language you can translate your content (Übersetzungen).

File: packages/plugins/i18n/admin/src/translations/de.json (hunk 27 above)
Comment (gu-stav) [reply packages/plugins/i18n/admin/src/translations/de.json:14]: I think we can leave this one out tbh.

**Comment by gu-stav** [comment]
@EarMaster I've committed a few suggestions @NicoHof made. Thanks to both of you for the work & discussion. A few suggestions are open - in case you'd like to get this into tomorrow's release we'd need to resolve them today.

**Comment by EarMaster** [comment]
I'm fine with how it is.

**Review by gu-stav — APPROVED** [review]
I think this is in a good state now. Thanks for the discussion and your contributions. 🌻

**Pull request merged**